yb-voyager/cmd/import.go

```diff
@@ -157,6 +158,8 @@ func registerImportSchemaFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&target.IgnoreIfExists, "ignore-exist", false,
 		"true - to ignore errors if object already exists\n"+
 			"false - throw those errors to the standard output (default false)")
+	cmd.Flags().BoolVar(&flagRefreshMViews, "refresh-mviews", false,
```

Comment:
can we just name this bool variable `refreshMViews` or `isRefreshMViews`

Reply:
I named it to have consistency among flag names as `--post-import-data` flag is also named as `flagPostImportData`.